### ISSUE TITLE
[BUG] 600MS Delay 😉

### ISSUE THREAD

**Issue by rahaaatul** [deltazefiro/Amarok-Hider#143]
# A little delay
Amarok shows it's true colors before hiding itself as a calendar 😅

# How it happens

https://github.com/deltazefiro/Amarok-Hider/assets/86149861/f09f59e7-0834-4ae3-a575-fa50c2fbd21d

# How to reproduce
Steps to reproduce the behavior / 复现异常的步骤:
1. Go to **Amarok**/**More Settings**/**Security**
2. Enable **App Disguise**
3. Restart Amarok

# Device Information
 - OS: Funtouch OS 11 Global (VIVO)
 - Android Version: 11
 - Amarok Version: 0.8.7a2

**Comment by deltazefiro** [comment]
Hi @rahaaatul ,
Do you use any modules that change the animation speed? I've tested on AOSP 11 but cannot reproduce.

**Comment by rahaaatul** [comment]
> Hi @rahaaatul , Do you use any modules that change the animation speed? I've tested on AOSP 11 but cannot reproduce.

Could this be the source of the issue?
![Screenshot_2024_0410_122353](https://github.com/deltazefiro/Amarok-Hider/assets/86149861/66204f72-81de-45e8-9c85-4943442f842f)

**Comment by deltazefiro** [comment]
> Could this be the source of the issue?

No, I'm afraid. The transition animation time between these two views is set to zero. Multiplying zero by .5 should not make any difference. Perhaps you tweak your system by some module to override animation time or change the behavior of the splash screen (i.e. [RestoreSplashScreen](https://github.com/GSWXXN/RestoreSplashScreen))?

**Comment by rahaaatul** [comment]
> > Could this be the source of the issue?
> 
> No, I'm afraid. The transition animation time between these two views is set to zero. Multiplying zero by .5 should not make any difference. Perhaps you tweak your system by some modules to override animation time or change the behavior of the splash screen (i.e. [RestoreSplashScreen](https://github.com/GSWXXN/RestoreSplashScreen))?

Unfortunately, I'm not rooted. So I don't use any modules.

**Comment by I21b** [comment]
> > Could this be the source of the issue?
> 
> No, I'm afraid. The transition animation time between these two views is set to zero. Multiplying zero by .5 should not make any difference. Perhaps you tweak your system by some modules to override animation time or change the behavior of the splash screen (i.e. [RestoreSplashScreen](https://github.com/GSWXXN/RestoreSplashScreen))?

If it's difficult to fix, maybe try changing the app's launch activity in the launcher.

**Comment by Omegaplexx** [comment]
Yeah, same issue here. The behaviour is the same as described above

- Samsung Galaxy S7
- LineageOS 17.1 (Android 10)
- Rooted (Magisk 27.0)
- Modules: LSPosed, Shamiko, Zygisk Next, AdGuard Certificate
- System animations speed is untouched (1x on all)
